kcl/provider.yaml

```diff
@@ -0,0 +1,11 @@
+# Install the ec2 provider, which contains CRDs and Controllers for EC2 resources
+# like VPCs and VM Instances.
+#
+# Installing this package will automatically pull in the upbound-family-aws provider
+# Upbound Marketplace link: https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v1.1.1
```

Comment:
```suggestion
# Upbound Marketplace link: https://marketplace.upbound.io/providers/upbound/provider-aws-ec2/v1.2.0
```

Reply:
this one should still be updated as suggested, right? it's intended to point to the marketplace listing of the provider in this file? (which is `provider-aws-ec2:v1.2.0`)